vm/systemSmartContracts/logs.go

```diff
@@ -0,0 +1,43 @@
+package systemSmartContracts
+
+import (
+	"math/big"
+
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+)
+
+func (d *delegation) createLogEntryForDelegate(
+	funcIdentifier string,
+	callerAddr []byte,
+	delegationValue *big.Int,
+	globalFund *GlobalFundData,
+	delegator *DelegatorData,
+	dStatus *DelegationContractStatus,
+	isNew bool,
+) *vmcommon.LogEntry {
+	numUsersWithCurrent := dStatus.NumUsers
+	if isNew {
+		numUsersWithCurrent++
+	}
+
+	activeFund := big.NewInt(0)
+	if len(delegator.ActiveFund) != 0 {
+		fund, err := d.getFund(delegator.ActiveFund)
+		if err != nil {
+			log.Warn("d.createLogEntryForDelegate cannot get fund", "error", err.Error())
```

Comment:
this is not log.Warn - this is critical error. return it as.

Reply:
refactored this part and kept the log.